src/_http.js

```diff
@@ -90,16 +90,14 @@ HttpClient.prototype.execute = function(method, path, body, query, options) {
   headers['X-Query-Timeout'] = queryTimeout
 
   var timeout
-  if (!signal && this._timeout) {
-    var abortController = new AbortController()
-    signal = abortController.signal
-    timeout = setTimeout(abortController.abort, this._timeout)
+  if (this._timeout) {
```

Comment:
I _think_ this will apply the same default query timeout to both queries and streams, right? If so, that's undesirable as streams are suppose to be long running handlers that don't timeout.

Reply:
not really. it's a timeout request to api POST /stream, so if this api doesn't respond in timeout, it will cancel the request. but as soon as request success, driver create stream